diff --git a/dist/index.mjs b/dist/index.mjs
index 7492f544a768161b3f9a79675354a24ffc4bb034..a992561a68add75e5b7867c55824ec9faef1bc6d 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -816,7 +816,6 @@ import EventEmitter4 from "eventemitter3";
 // src/relay/index.ts
 import EventEmitter3 from "eventemitter3";
 import { relayInit } from "nostr-tools";
-import "websocket-polyfill";
 import debug from "debug";
 var NDKRelayStatus = /* @__PURE__ */ ((NDKRelayStatus2) => {
   NDKRelayStatus2[NDKRelayStatus2["CONNECTING"] = 0] = "CONNECTING";